test/test.jl

```diff
@@ -122,6 +125,14 @@ str = sprint(show, fails[8])
 @test contains(str, "Evaluated: isapprox(0.0, -Inf)")
 
 str = sprint(show, fails[9])
+@test contains(str, "Expression: isapprox(1 / 2, 2 / 1, atol=1 / 1)")
+@test contains(str, "Evaluated: isapprox(0.5, 2.0; atol=1.0)")
```

Comment:
Note that the evaluated expression currently will always display a `;` when keywords are used.